nav2_map_server/test/component/test_map_server_node.cpp

```diff
@@ -24,6 +24,8 @@
 #include "nav2_map_server/map_server.hpp"
 #include "nav2_util/lifecycle_service_client.hpp"
 #include "nav2_msgs/srv/load_map.hpp"
+using namespace std::chrono_literals;
+using namespace rclcpp;  // NOLINT
```

Comment:
I do not think, we need to be in `rclcpp` namespace. Everything is seems already addressed starting from rclcpp, e.g.:
```
auto parameters_client = std::make_shared<rclcpp::SyncParametersClient>(node_, "map_server");
```
so this is unnecessary